bundles/net.tourbook/src/net/tourbook/ui/views/tourBook/TVITourBookYearSub.java

```diff
@@ -270,6 +286,7 @@ protected void fetchChildren() {
                final long dbDrivingTime         = tourItem.colTourDrivingTime = result.getLong(6);
                tourItem.colAltitudeUp           = result.getLong(7);
                tourItem.colAltitudeDown         = result.getLong(8);
+               tourItem.colAvgAltitudeChange    = result.getLong(84);
```

Comment:
reading field 84 should not be out of the sequence.

`colTemperature_Min/Max` is also read at the end and not where `colTemperature_Avg` is located, in this case the sequence is more important than the grouping

Reply:
Done (see PR)